signatures/gemss/RedGemss/bench/additional_192_skylake.yaml

```diff
@@ -0,0 +1,6 @@
+platform: Skylake
+timings:
+  unit: cycles
+  gen: 277000000
+  enc|sign: 13100000
+  dec|vrfy: 518000
```

Comment:
Are you using an old version or something? I see 266 000 000 / 11 800 000 / 453 000